server/controllers/userController.js

```diff
@@ -1,21 +1,17 @@
 /* eslint-disable require-jsdoc */
 /* eslint-disable class-methods-use-this */
 import models from '../db/models';
-import auth from '../helpers/auth'
+import auth from '../helpers/auth';
+import Mailer from '../helpers/mailer';
+import util from '../helpers/utilities';
 
 const UserController = {
     async signUp(req, res) {
-        models.User.findOne({
-            where: { email: req.body.email }
-          })
-          .then((foundUser) => {
-            if (foundUser) {
-              return res.status(409).send({
-               status: res.statusCode,
-               message: 'email address exist already', 
-              });
+          const user = await models.Users.findOne({where: {email: req.body.email}});
+            if(user){
+                //make changes to jude's work on naming convention change errorstatut to errorStatus
+                return util.errorStatus(res, 409, 'email address exist already');
             }
-
             const hashPassword = auth.hashPassword(req.body.password);
 
             const user = {
```

Comment:
@OvieMudi @encodedBicoding Guys, pls check lines 10 and 17. const user is declared twice within the same scope

Reply:
This has been taken care of.